azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_babylonawsassumerole.go

```diff
@@ -0,0 +1,52 @@
+package serviceendpoint
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/serviceendpoint"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/tfhelper"
+)
+
+func ResourceServiceEndpointBabylonAWSAssumeRole() *schema.Resource {
```

Comment:
Would it make sense for this just to be called serviceendpoint_babylonawsiam to match the extension: https://github.com/babylonhealth/azure-extensions/blob/master/vss-extension.json#L54 ?

Makes less assumptions about what the service connection is to be used for